app/config/security.yml

```diff
@@ -65,6 +65,12 @@ security:
             simple_preauth:
                 authenticator: apikey_authenticator
 
+        api_check_notifications:
+            pattern: ^.*/api/notifications/get.json
+            stateless: true
+            simple_preauth:
+                authenticator: apikey_authenticator
+
```

Comment:
We may want to use symfonys apikey system to log the user in during the checks for notifications, since the app probably can't deal with session cookies.